src/n-dhcp4-client.c

```diff
@@ -324,6 +337,13 @@ NDhcp4CEventNode *n_dhcp4_c_event_node_free(NDhcp4CEventNode *node) {
         case N_DHCP4_CLIENT_EVENT_EXTENDED:
                 node->event.extended.lease = n_dhcp4_client_lease_unref(node->event.extended.lease);
                 break;
+        case N_DHCP4_CLIENT_EVENT_LOG:
+                if (_c_unlikely_(node->event.log.is_static_message)) {
```

Comment:
Does `is_static_message` mean that the message string is static or that the event node itself is static (or both)?

Reply:
renamed field.